### PR TITLE
Ch2 issue74 - Uploaded the document for Chapter 2 (Sprint 4)

### LINKED ISSUE #74: How might we simplify and translate technical jargon for a non-technical audience?
**Section Outline**
Please list the key points of content you intend to cover for this section
 
 
**Definition of Done**
- [x] Completed `2-proj-mgmt-tech-lead/translate-technical-jargon.md`
- [x] Verified against [self-review checklist](https://github.com/aisingapore/ai-practitioner-handbook/blob/main/REVIEWING.md)
- [x] Added section to [table of contents](https://github.com/aisingapore/ai-practitioner-handbook/blob/main/_toc.yml)
- [x] Page builds and displays correctly on local machine

### PR DESCRIPTION
**Pull request by shafirahmad**
Please review.

### CODE REVIEW

**Comment by chuawjk** [comment]
https://oss.gitlab.aisingapore.net/aisg-handbook/ai-practitioner-handbook/book/2-proj-mgmt-tech-lead/translate-technical-jargon.html

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# How might we simplify and translate technical jargon for a non-technical audience?
+Contributor: Shafir Ahmad, Senior AI Technical Consultant
+
+Communicating with people who are not actively involved in your project can present a challenge. Technical staff such as engineers and developers find it difficult to communicate complex technical concepts to clients, senior management and customers alike. It might even look like you’re speaking different languages.
+
+The fact is that many of these stakeholders you communicate with will understand the big picture and the high level concepts, but not the intricate engineering details. It is better to keep these details for a more technically inclined audience. 
+
+Here are some tips you can use to reach an understanding between yourself and non-technical stakeholders.
+
+## 1. Know your audience
```
Comment (kevinosl) [inline book/2-proj-mgmt-tech-lead/translate-technical-jargon.md:10]: I suggest calling out an additional scenario - sometimes, there could be a varying level of knowledge among the audience. In such cases, what is our suggestion? (personally I suggest tailoring to somewhere near the 'lowest common denominator' while at the same time pre-empting the rest of the audience to bear with me if they already know some of the information)

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## 2. Reduce or Eliminate Tech Jargon
+
+Start eliminating tech jargon and acronyms from your conversations with the stakeholders. Certain terms may not make any sense to the stakeholder, and others may hold a different meaning from the one you commonly use (e.g. NLP = Neuro Linguistic Programming vs NLP = Natural Language Processing). Although it may seem like there is no way to communicate the details without using the jargon you are used to, there are ways to mitigate this. Consider the use of metaphors, analogies and storytelling techniques that will help your audience understand the material.
```
Comment (kevinosl) [inline book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: I've added this additional blurb:
>>>>>
_Try translating technical terms into their equivalents in everyday language. 
Pro tip: it helps if you understand the technicalities deeply!
Here's an example if communicating with a non-technical decision-maker:
Tech jargon: "we will ship an end-to-end pipeline for you to retrain the model upon data drift in your production system."
Everyday language: "we will share the code and instructions with your technical team to enable them to refresh the model after the solution has been implemented and you have gathered enough new data."_

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## 2. Reduce or Eliminate Tech Jargon
+
+Start eliminating tech jargon and acronyms from your conversations with the stakeholders. Certain terms may not make any sense to the stakeholder, and others may hold a different meaning from the one you commonly use (e.g. NLP = Neuro Linguistic Programming vs NLP = Natural Language Processing). Although it may seem like there is no way to communicate the details without using the jargon you are used to, there are ways to mitigate this. Consider the use of metaphors, analogies and storytelling techniques that will help your audience understand the material. 
+
+## 3. Provide background information and education
```
Comment (kevinosl) [inline book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: Suggest calling this ```Provide background information, context and education```

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+Do not assume that the stakeholder knows or does not know certain concepts or terms. However, do assume that they are intelligent and will be able to understand if given an adequate explanation. Do translate key jargon and acronyms that are important to the discussion in a way that makes it simple for the stakeholder to understand. Metaphors and analogies are a good way to translate a technical concept into a more familiar context. It could be useful to give a glossary and definitions of some necessary terms.
```
Comment (kevinosl) [inline book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: In addition to not assuming the stakeholder knows concepts or terms, it's also wise not to assume they have the full picture or context of why we're having this discussion. So I usually find it beneficial to start off a discussion by quickly recapping where we're at and reminding the audience of the objective of the current discussion. Maybe we can add this point in.

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## 4. Use visuals to illustrate technical concepts
+
+While written words and verbal speech are effective ways of communication, working to make the concepts visual can make it more effective. You may have heard the old adage “A picture is worth a thousand words”. Research suggest that adding visuals can help increase memory recall to 65% versus 10% from just hearing it. With the use of creative illustrations such as flowcharts, process diagrams, demos, models and architecture diagrams, stakeholders can learn and understand the most technical concepts from these visual aids. Remember to tailor it to your audience as a high level executive may not need an in depth understanding of an architecture diagram, but needs an overview.
```
Comment (kevinosl) [inline book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: Suggest adding a piece of advice to keep the visuals as uncluttered and easy to understand as possible (avoiding information overload). We can show a good and bad example side by side (which is in line with our own advice of using visuals!)

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+With just a bit of thought and planning, you can make the discussion more engaging and understandable for the participants. You may need to arrange for additional meetings to provide the participants with the level of depth of understanding they require.
```
Comment (kevinosl) [inline book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: Suggest adding another pro tip to round it off: an excellent way to practice is to try explaining some of your work to your spouse or friend! As the saying goes, “If you can't explain it simply, you don't understand it well enough.”

**Comment by kevinosl** [comment]
@shafirahmad I've submitted my review. Thanks!

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md (hunk 4 above)
Comment (shafirahmad) [reply book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: This is not really part of translating technical terms for a non-technical audience. I feel this is more on general client/meeting management, which we do not have a chapter currently

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md (hunk 3 above)
Comment (shafirahmad) [reply book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: OK

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md (hunk 2 above)
Comment (shafirahmad) [reply book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: I've added this example

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md (hunk 1 above)
Comment (shafirahmad) [reply book/2-proj-mgmt-tech-lead/translate-technical-jargon.md:10]: I have added this, however I think this is general meeting etiquette and not specific to getting a technical point across.

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md (hunk 5 above)
Comment (shafirahmad) [reply book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: Added (without example)

File: book/2-proj-mgmt-tech-lead/translate-technical-jargon.md (hunk 6 above)
Comment (shafirahmad) [reply book/2-proj-mgmt-tech-lead/translate-technical-jargon.md]: OK

**Pull request closed without merging**